src/Files.Uwp/Views/MainPage.xaml.cs

```diff
@@ -75,9 +79,38 @@ public MainPage()
             ToggleCompactOverlayCommand = new RelayCommand(ToggleCompactOverlay);
             SetCompactOverlayCommand = new RelayCommand<bool>(SetCompactOverlay);
 
+            if (SystemInformation.Instance.TotalLaunchCount >= 10 & Package.Current.Id.Name == "49306atecsolution.FilesUWP" && !UserSettingsService.ApplicationSettingsService.WasPromptedToReview)
```

Comment:
We should really create a separate service for this

Reply:
Perhaps we can combine the logic and the setting in a "ReviewPromptService" or similar.

Reply:
Definitely, the question is if we can get this in as is for now given the high impact it'll have in the store?